central/complianceoperator/v2/report/manager/manager_impl.go

```diff
@@ -29,7 +32,9 @@ type reportRequest struct {
 }
 
 type managerImpl struct {
-	datastore            scanConfigurationDS.DataStore
+	datastore     scanConfigurationDS.DataStore
```

Comment:
nit: now that we are using multiple datastores should probably rename it to something like `scanConfigDataStore` for clarity.